src/numerics.jl

```diff
@@ -0,0 +1,326 @@
+using UnPack
+using SparseArrays
+using LinearAlgebra
+
+include("Klimaparameter.jl")
+
+#Mapping functions
+##################
+function index2d(k,nx)
+    j = floor(Int,(k-1)/nx)+1
+    # TODO: add consistency check with ny?
+    return k-(j-1)*nx,j 
+end
+
+function index1d(i,j,nx)
+    # TODO: add consistency check with ny?
+    return i+(j-1)*nx
+end
+
+struct mesh_t
```

Comment:
Convention in Julia is to use `CamelCase` for types and `snake_case` for functions and variables. I recommend to stick to that since it makes the code easier to read.